agent_event_publisher_http/Cargo.toml

```diff
@@ -12,18 +12,19 @@ agent_verification = { path = "../agent_verification" }
 anyhow = "1.0"
 async-trait.workspace = true
 cqrs-es.workspace = true
-reqwest = { version = "0.11", features = ["json"] }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Could we use [rustls](https://crates.io/crates/rustls) instead of `openssl`?

Reply:
I had some problems with selecting the right TLS because it didn't seem to work together with `postgres-es` and/or `sqlx` before. But I had another go now trying something different and it does work now, so thanks for pointing this out :+1: